main.py

```diff
@@ -192,9 +195,12 @@ def main():
 
                 # evaluate
                 if total_steps % args.eval_freq == 0:
-                    eval_reward = evaluate_policy(gac, eval_env, args.eval_episodes)
+                    eval_rewards = evaluate_policy(gac, eval_env, args.eval_episodes)
+                    eval_reward = sum(eval_rewards) / args.eval_episodes

```

Comment:
Just append the average eval rewards here to a list and use a dict format to save this as a json in a file.